http-handler/src/main/java/com/flipkart/poseidon/handlers/http/impl/SinglePoolHttpTaskHandler.java

```diff
@@ -112,7 +111,16 @@ public void init(TaskContext taskContext) throws Exception {
             throw new Exception("Invalid pool name specified");
         }
 
-        // create the pool object       
+        // Calculate the maxConnections from poolSize params.
+        int maxConnections = 10; // default as per TaskHandlerExecutorRepository.DEFAULT_THREAD_POOL_SIZE
+        if(getConcurrentPoolSizeParams() != null && !getConcurrentPoolSizeParams().isEmpty()) {
```

Comment:
Yes, this is documented in phantom at https://github.com/Flipkart/phantom/blob/2217d365c7317195b70b5d310826310f1254ce20/task/src/main/java/com/flipkart/phantom/task/impl/HystrixTaskHandler.java.
So ideally, concurrentPoolSizeParams is for pool name level and 1 pool can have multiple commands.
commandPoolSizeParams overrides concurrentPoolSizeParams to allow throttling at command level.
In our case of SPHTH, We have a single pool and single command. We dont use commandPoolSizeParams, but still for future, making the change.